docs/en/observability/logs-stream.asciidoc

```diff
@@ -143,10 +143,246 @@ If you're not seeing your log files in {kib}, check the following in the `elasti
 If you're still running into issues, see {fleet-guide}/fleet-troubleshooting.html[{agent} troubleshooting] and {fleet-guide}/elastic-agent-configuration.html[Configure standalone Elastic Agents].
 
 [discrete]
-[[logs-stream-whats-next]]
-== What's next?
+[[logs-stream-enhance-logs]]
+== Get the most out of your log data
 
-For more information on deploying and managing logs in Elastic Observability, see the following links:
+This section shows you how to get the most out of your log data after sending it to {es} by extracting structured fields from your unstructured log data. Extracting structured fields makes it easier to search, analyze, and filter your log data. 
```

Comment:
> This section shows you how to get the most out of your log data after sending it to {es} by extracting structured fields from your unstructured log data.

I can't explain why, but I don't like this sentence. Here's a suggestion. You don't have to take it verbatim but maybe think more about how we open this section.

```
== Get the most out of your log data

Make your logs more useful by extracting structured fields from your unstructured log data.
Extracting structured fields makes it easier to search, analyze, and filter your log data. 
```